ecosystem/node-checker/src/main.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+// For use while we're developing.
+#![allow(dead_code)]
+#![allow(unused_imports)]
+#![allow(unused_variables)]
+
+mod args;
+mod build_evaluators;
+mod metric_collector;
+mod metric_evaluator;
+mod public_types;
+mod runner;
+
+use anyhow::{anyhow, bail, Context, Result};
+use args::Args;
+use build_evaluators::build_evaluators;
+use clap::Parser;
+use log::{debug, info};
+use metric_collector::{MetricCollector, ReqwestMetricCollector};
+use metric_evaluator::MetricsEvaluator;
+use poem::{
+    handler, http::StatusCode, listener::TcpListener, Error as PoemError, Result as PoemResult,
+    Route, Server,
+};
+use poem_openapi::{payload::Json, OpenApi, OpenApiService};
+use public_types::{CompleteEvaluation, NodeUrl};
+use reqwest::Client as ReqwestClient;
+use runner::{BlockingRunner, BlockingRunnerArgs, Runner};
+use std::{collections::HashSet, hash::Hash, path::PathBuf, sync::Arc};
+use url::Url;
+
+// TODO: Replace this with the real frontend, or perhaps an error handler if we
+// decide to route the frontend to just a static hoster such as nginx.
+#[handler]
+fn root() -> String {
+    "Hello World!".to_string()
```

Comment:
nit: can we replace this with a TODO? Just in case someone starts pinging/running it?